nuttx/arch/arm/src/stm32/Kconfig

```diff
@@ -1235,156 +1926,166 @@ if STM32_ADC1
 menu "ADC1 Device Initialization"
 
 config STM32_ADC1_CHAN0_INIT
-	bool "ADC1 Channel 0"
+	bool "ADC1 Channel 0 (IN1-PC0)"
```

Comment:
Sorry, maybe I am not following how the Kconfig is arranged.

STM32_ADC1_CHAN0_INIT is shared by all flavors of STM32 chip, right? Not just STM32L4xx, but also STM32Fxxx, etc. In these cases, is PC0 always the pin?

STM32_ADC1_CHAN0_INIT is the config for the ADC1, channel 0 initialization function, but that function might physically be routed out on different pins depending on the flavor of the STM chip.